src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -28,12 +28,16 @@ const propTypes = {
      */
     toggleReaction: PropTypes.func.isRequired,
 
+    /** A bool indicating if the current report action has any errors */
+    hasReportActionErrors: PropTypes.bool,
```

Comment:
NAB: It's a boolean, so it's better to name it with the `should` or `is` prefix.